addons/html_editor/static/src/others/dynamic_placeholder/dynamic_placeholder.js

```diff
@@ -0,0 +1,92 @@
+import { _t } from "@web/core/l10n/translation";
+import { Plugin } from "@html_editor/plugin";
+
+export class DynamicPlaceholderPlugin extends Plugin {
```

Comment:
Not testable yet... ?

Reply:
Yes I have to add some unit test indeed. I think it should be doable if i provide a "fake" model for the reference field.